src/bantz/data/auto_linker.py

```
@@ -0,0 +1,271 @@
+                return 0
+        return 0
+
+    # ── email linker ──
+
+    async def _link_email(self, data: Dict[str, Any]) -> int:
+        """Extract entities from a Gmail result.
+
+        Expected keys: from, to, cc, subject, message_id, snippet, date
+        """
+        edges_created = 0
+        now = time.time()
```

Ruff (0.15.0)

[error] Local variable `now` is assigned to but never used

Remove assignment to unused variable `now`

(F841)